shim/handler.go

```diff
@@ -232,8 +225,7 @@ func (h *Handler) handleTransaction(msg *peer.ChaincodeMessage) (*peer.Chaincode
 		return nil, fmt.Errorf("failed to marshal response: %s", err)
 	}
 
-	err = h.sendBatch(msg.ChannelId, msg.Txid)
-	if err != nil {
+	if err := stub.FinishWriteBatch(); err != nil {
```

Comment:
you can only leave equals here

```
if err = stub.FinishWriteBatch(); err != nil {
```

Reply:
I prefer to have an err specific to the if block

Reply:
I disagree.

You're just creating an extra variable here, which you return immediately. 
Show me an example of how a new variable could be better here?